ostd/src/mm/vm_space.rs

```diff
@@ -186,6 +239,11 @@ impl VmSpace {
         // SAFETY: The memory range is in user space, as checked above.
         Ok(unsafe { VmWriter::<Fallible>::from_user_space(vaddr as *mut u8, len) })
     }
+
+    /// Get the current [`VmMappingPolicy`] attached to this address space.
+    pub fn vm_mapping_policy(&self) -> &dyn VmMappingPolicy {
+        &*self.vm_mapping_policy
+    }
```

Comment:
Why do you use an accessor here? I think it might be reasonable since it limits operations (mainly it makes it impossible to clone the Arc), but I'm curious what your reason was.

Reply:
We don't have that nailed down at all. :-) This all makes sense.